Uchu.StandardScripts/AvantGardens/MaelstromSample.cs

```diff
@@ -1,24 +1,43 @@
+using System.Linq;
+using System.Threading.Tasks;
 using Uchu.World;
 using Uchu.World.Scripting.Native;
 using Uchu.Core.Resources;
 
 namespace Uchu.StandardScripts.AvantGardens
 {
     /// <summary>
-    /// Native implementation of scripts/02_client/map/ag/l_ag_maelstrom_sample.lua
+    /// Script to show/hide maelstrom samples based on whether the player has the relevant mission
     /// </summary>
-    [ScriptName("l_ag_maelstrom_sample.lua")]
-    public class MaelstromSample : ObjectScript
+    [ZoneSpecific(1100)]
+    public class MaelstromSample : NativeScript
```

Comment:
What was the purpose of making this `ZoneSpecific` again?

Reply:
The ScriptName it used refers to a client script. After the changes on this branch, these don't get loaded anymore. No server script name is attached to these objects, so I've reverted this script to an older NativeScript implementation.

Reply:
(Just going to use this now since there is 2 of the same comment)
Any thoughts about an `ObjectSpecific` or `LotSpecific` attribute so we don't have extra, duplicate event handlers?